storage/rocksdb/ha_rocksdb.cc

```diff
@@ -5608,11 +5608,9 @@ int ha_rocksdb::read_range_first(const key_range *const start_key,
       DBUG_RETURN(HA_ERR_LOCK_DEADLOCK);
     }
 
-    MYSQL_TABLE_IO_WAIT(m_psi, PSI_TABLE_FETCH_ROW, active_index, 0, {
-      result =
-          index_read_map_impl(table->record[0], start_key->key,
-                              start_key->keypart_map, start_key->flag, end_key);
-    })
+    result =
```

Comment:
I think it is in the sense that it might be over counting certain index operations, but, I am far from an expert here. The upstream commit that introduces this was https://github.com/facebook/mysql-5.6/commit/aa89833f8fcef6ce7346ae94db9a1463e069b7ec and comes from Sergei P but I think it was wrong originally and is definitely wrong in 5.7 considering that the macros are hidden in handler.cc